Jenkinsfile

```diff
@@ -12,10 +12,13 @@ pipeline {
 		maven 'apache-maven-latest'
 		jdk 'openjdk-jdk21-latest'
 	}
+	environment {
+		NON_MODULAR_JAVA_HOME = "${tool(type: 'jdk, name: 'temurin-jdk8-latest')}"
+	}
```

Comment:
In order to pick this change up in a verification build (and make all tests pass) a Committer of JDT would have to toe _Replay_ the build with this change.